tests/TestingTraitTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace RonasIT\Support\Tests;
+
+use RonasIT\Support\Exceptions\UnknownRequestMethodException;
+use RonasIT\Support\Traits\ExceptionsTestTrait;
+use RonasIT\Support\Traits\TestingTrait;
+
+class TestingTraitTest extends TestCase
+{
+    use TestingTrait;
+
+    public function testAssertExceptionThrew(): void
+    {
+        $this->assertExceptionThrew(UnknownRequestMethodException::class, "Unknown request method 'unsupported'");
+
+        throw new UnknownRequestMethodException('unsupported');
```

Comment:
that's strange, test expecting exception with the message "Unknown request method 'unsupported'" in strict mode, so it should passed only in case exception with exactly this message will be trow, but in fact the exception with the message 'unsupported' had been passed and test had been completed successfully.

Please check why it happened